doc/grid_ufuncs.rst

```diff
@@ -0,0 +1,443 @@
+.. _grid_ufuncs:
+
+Grid Ufuncs
+-----------
+
+Concept of a Grid Ufunc
+~~~~~~~~~~~~~~~~~~~~~~~
+
+In short, a "grid ufunc" is a generalisation of a `numpy generalized universal function`_ to include the xGCM Axes and Positions of input and output variables.
+
+If you are not already familiar with numpy generalised universal functions (hereon referred to as "numpy ufuncs"),
+then here is a quick primer.
+
+.. dropdown:: **Primer on numpy generalized universal functions**
+
+    Content about numpy...
```

Comment:
Do we want to fill this in before merging? Or otherwise just link out to the numpy docs.

Reply:
Oh oops - I was originally going to write out my own explanation here but actually I think a link to the numpy docs should be sufficient.